helpers/src/host_information.rs

```diff
@@ -57,10 +56,8 @@ fn server_type() -> Result<String> {
     debug!("Gathering server type");
     let path = Path::new("/sys/class/dmi/id/product_name");
```

Comment:
WHere is this function called? (my search didn't provide useful leads). Anyway, I am not certain if the paths should be hardcoded. Add a todo/note perhaps?

Reply:
It's used here: https://github.com/Comcast/Bynar/blob/bug_fixes/src/main.rs#L116.  I think that's a special file on linux that gets created if the dmi information is read correctly.  There's a partial implementation of this information here: https://gitlab.redox-os.org/redox-os/dmi/blob/master/src/lib.rs.  I'll check out that code again and see if it does what we need.